lib/lita/extensions/confirmation.rb

```diff
@@ -24,6 +24,7 @@ def call
           message.reply(
             I18n.t(
               "lita.extensions.confirmation.request",
+              prefix: @robot.alias ? @robot.alias : "#{@robot.mention_name} ",
```

Comment:
That happened when running the tests? It might be the adapter loading code in the robot failing to find an adapter. Make sure at least the shell adapter is registered in a before block. That's bitten me a few times and something I think should be improved for testing usability.